Resources/Lang/en_US.json

```diff
@@ -3111,6 +3114,16 @@
   "VultureMaxEatenInOneRound": "Maximum eaten bodies possible per round",
   "VultureCooldownUp": "Eat Cooldown finished",
 
+  "PossessCD": "Possess Cooldown",
+  "MaxPossessions": "Max Possessions",
+  "PossessionDuration": "Possession Duration",
+  "GhastlySpeed": "Ghastly Speed",
+  "CannotPossessTarget": "Couldn't Possess Target",
+  "ChooseTarget": "Now: Choose Target",
+  "NoMorePossess": "You've ran out of possessions!'",
+  "NotUrTarget": "That is not your target",
+  "YouvePosses":  "You've Been Possessed!",
```

Comment:
Next time, make the settings so that it is clear what role they belong to

For example:
`GhastlyPossessCD`
`GhastlyMaxPossessions`

Because crowdin translators don't see the context

Reply:
ngl I tought they didn't even see that